web/src/app/services/ServiceRouter.js

```diff
@@ -31,66 +32,103 @@ const query = gql`
   }
 `
 
-export default class ServiceRouter extends React.PureComponent {
-  renderList = () => (
-    <SimpleListPage
-      query={query}
-      variables={{ input: { favoritesFirst: true } }}
-      mapDataNode={n => ({
-        title: n.name,
-        subText: n.description,
-        url: n.id,
-        isFavorite: n.isFavorite,
-      })}
-      createForm={<ServiceCreateDialog />}
-      createLabel='Service'
-    />
-  )
+export default function ServiceRouter() {
+  const searchParam = useSelector(searchSelector) // current total search string on page load
+  const dispatch = useDispatch()
+  const setSearchParam = value => dispatch(setURLParam('search', value))
 
-  renderDetails = ({ match }) => (
-    <ServiceDetails serviceID={match.params.serviceID} />
-  )
-  renderAlerts = ({ match }) => (
-    <ServiceAlerts serviceID={match.params.serviceID} />
-  )
-  renderKeys = ({ match }) => (
-    <IntegrationKeyList serviceID={match.params.serviceID} />
-  )
-  renderHeartbeatMonitors = ({ match }) => (
-    <HeartbeatMonitorList serviceID={match.params.serviceID} />
-  )
-  renderLabels = ({ match }) => (
-    <ServiceLabelList serviceID={match.params.serviceID} />
-  )
+  // grab key and value from the search param, if at all
+  let key = null
+  let value = null
+  if (searchParam.includes('=')) {
+    const searchSplit = searchParam.split(/(!=|=)/)
+    key = searchSplit[0]
+    value = searchSplit[2] // [1] being != or =
```

Comment:
The value can contain `=` so we should join the rest of the match so it doesn't get lost:
```js
value = searchSplit.slice(2).join('')
```